### ISSUE TITLE
Dark Theme issue on Details page

### ISSUE THREAD

**Issue by coliver** [alexandersokol/sd-model-organizer#6]
When viewing the details for a item on the dark theme, some of the rows are 'disappearing' because of the white text on the white background.

Light:
![image](https://github.com/alexandersokol/sd-model-organizer/assets/632788/e13e4458-81f0-4672-9a00-2dfc06cbbc3b)

Dark:
![image](https://github.com/alexandersokol/sd-model-organizer/assets/632788/2909b2a9-4603-4782-adc6-82063fab449b)

**Comment by alexandersokol** [comment]
Hello!
Seems you are adding dark theme param to url , but extension works only with `--theme dark` argument while starting webui, can you try this way?

**Comment by coliver** [comment]
> Hello! Seems you are adding dark theme param to url , but extension works only with `--theme dark` argument while starting webui, can you try this way?

Yes, I will try adding that. 

I only used the param in the url to take those screenshots. I was not using the param until i needed the light theme screenshot. My system is defaulting to dark. 

(Later)

Ok, I've added the the `--theme dark` arg to my webui-user.bat. It looks good:

![image](https://github.com/alexandersokol/sd-model-organizer/assets/632788/f1d0b0f4-db37-4d85-b767-b4495b8c736d)

After adding the `--theme dark` arg, it added the param (/?__theme=dark) by itself.

Thanks for the quick reply.
